### PR TITLE
Adding chunky3d

### PR DESCRIPTION
**Pull request by tgandor**
This is a pure-python library recently released by K3D-team.
Github URL: https://github.com/K3D-tools/chunky3d
PyPI: https://pypi.org/project/chunky3d/

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tgandor** [comment]
@artur-trzesiok - please comment here if you agree to being a maintainer of this feedstock.

**Comment by artur-trzesiok** [comment]
Sure!

File: recipes/chunky3d/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - setuptools
```
Comment (scopatz) [inline recipes/chunky3d/meta.yaml]: you don't need this build section at all

File: recipes/chunky3d/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - scipy
+    - simpleitk
+    - vtk
+    
+
+test:
+  imports: []
+  # TODO: make `test` a package, then probably:
+  #  - test.test_sparse
+  #  - test.test_sparse_func
```
Comment (scopatz) [inline recipes/chunky3d/meta.yaml]: you should have some kind of real test

File: recipes/chunky3d/meta.yaml (hunk 1 above)
Comment (tgandor) [reply recipes/chunky3d/meta.yaml]: thanks, fixed.

File: recipes/chunky3d/meta.yaml (hunk 2 above)
Comment (tgandor) [reply recipes/chunky3d/meta.yaml]: I don't fully understand the `run_tests.py` - should I write it myself and include in the recipe, or is it auto-generated and I need only to specify the modules (containing tests) which will be imported?

If the tests need to be importable, I need to make the test/ subdirectory of my sdist package a module (it's a raw folder now). For that I probably need to publish a next version on PyPI.

Or maybe - the tests are running in the unpacked source directory, and I just need to write `command: "{{ PYTHON }} test/test_sparse.py && {{ PYTHON }} test/test_sparse_func.py`?

Comment (tgandor) [reply recipes/chunky3d/meta.yaml]: OK, I found this: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-section
I think I'm on my way to fixing this.

**Comment by tgandor** [comment]
OK, I've solved it without the need to publish a new release on PyPI. The tests are running, also, our library isn't `noarch`, it doesn't work on Py 2.7 (not only on Windows; as thought previously).
I think we're probably good to go now.

File: recipes/chunky3d/meta.yaml (hunk 2 above)
Comment (scopatz) [reply recipes/chunky3d/meta.yaml]: Also you can look at the exampple or any of the feedstock repos in this org

**Comment by tgandor** [comment]
Thanks for all the suggestions! 

Finally the checks now pass. Effectively, this limits the platforms to Python 3.x on Linux and OS X for the time being. We're working on making it lighter (https://github.com/K3D-tools/chunky3d/issues/3).

I think we're ready, please review!

**Comment by scopatz** [comment]
Thanks!

**Comment by tgandor** [comment]
Great. Many thanks for all the help and merging.

**Pull request merged**